src/Tempest/CommandBus/src/CommandBusConfig.php

```diff
@@ -14,6 +15,9 @@ public function __construct(
 
         /** @var array<array-key, class-string<\Tempest\CommandBus\CommandBusMiddleware>> */
         public array $middleware = [],
+
+        /** @var class-string<\Tempest\CommandBus\CommandRepository> $commandRepositoryClass */
+        public string $commandRepositoryClass = FileCommandRepository::class,
```

Comment:
If we want a different driver in production, we'll need to specify the FQCN of the repository in environment variables so we can do this:

```php
return new CommandBusConfig(
    commandRepositoryClass: env('COMMAND_REPOSITORY'),
);

// .env
COMMAND_REPOSITORY="Tempest\CommandBus\AsyncCommandRepositories\FileCommandRepository"
```

This is not ideal. How about abstracting this away? Might be out-of-scope for this PR, but improving on this later will be a breaking change too.

```php
use Tempest\CommandBus\Driver;

return new CommandBusConfig(
    asyncCommandDriver: Driver::from(env('COMMAND_DRIVER')),
);
```

Reply:
Gave it some thought, it's not as trivial as I imagined: if we want a mapping somewhere, we need to know of all available `CommandRepository` implementations. Which means adding discovery for it, because we want to support project and third part repository implementations as well. That's a lot of overhead. 

This is actually a problem not limited to this PR, so I'm going to keep it as-is for now, and we need to think about ways to improve it.